src/render/styles/knotx.css.less

```diff
@@ -289,9 +294,19 @@ pre {
   
   ul {
     padding: 0;
-    margin: 0;
-    list-style: none;
-    
+    margin: 0 0 10px 10px;
```

Comment:
It might affect "Recent Posts" right column section on /blog page - check it.
If you want to apply style for lists on blog post, rather start from `.post` css class
```
.post {
  ul {
   ....
  }
}
```